### DISCUSSION TITLE
file_log读取多个文件夹数据

### DISCUSSION THREAD

**Discussion by q121102302** [alibaba/ilogtail#803] [Help] [answered]
版本：1.3.1
<img width="755" alt="image" src="https://user-images.githubusercontent.com/101638329/232977959-2e6df22f-9524-483e-bfbb-02b28d9fc50b.png">
假如我/mnt/logs下面有 a b c d四个子文件夹 我只需要拿到 a和b的应该怎么写呢？
我设置了多个file_log的方式 但是看日志 就只打了一个项目的

**Comment by messixukejia** [comment, marked as answer]
两种方式：
1、因为一个配置目前只能支持一个file类型，所以可以把配置拆成多个文件。
2、LogPath是支持嵌套的，如果子目录是固定的，可以配制成/home/logs即可。

文档说明的不是太清晰，欢迎帮忙补充下文档
